pyccl/background.py

```diff
@@ -108,8 +109,28 @@ def comoving_angular_distance(cosmo, a):
         float or array_like: Comoving angular distance; Mpc.
     """
     cosmo.compute_distances()
-    return _vectorize_fn(lib.comoving_angular_distance,
-                         lib.comoving_angular_distance_vec, cosmo, a)
+    return _vectorize_fn(lib.angular_distance,
+                         lib.angular_distance_vec, cosmo, a)
```

Comment:
Why did these calls change? I think that is a bug.

Reply:
Ah yes, fixed